README.md

```diff
@@ -174,16 +174,24 @@ version is the image's digest.
 * `pull_tag`: *Optional.*  **DEPRECATED. Use `get` and `load` instead.** Default
   `latest`. The tag of the repository to pull down via `pull_repository`.
 
-* `tag`: *Optional.* The value should be a path to a file containing the name
-  of the tag.
+* `tag_name`: *Optional.* The tag. This takes precedence over `tag_file`. Will be prefixed/suffixed if those params are present.
+
+* `tag`: **DEPRECATED. Use `tag_file` instead**
+* `tag_file`: *Optional.* The value should be a path to a file containing the name
+  of the tag. Will be prefixed/suffixed if those params are present.
 
 * `tag_prefix`: *Optional.* If specified, the tag read from the file will be
   prepended with this string. This is useful for adding `v` in front of version
   numbers.
 
+* `tag_suffix`: *Optional.* Adds a suffix to the tag. Useful for things like `-$BUILD_ID`
```

Comment:
Does `-$BUILD_ID` actually work? I don't see where it'd be interpolated.

Reply:
this is talked about here: